### ISSUE TITLE
Bug: Local development instructions for MacOS are incomplete

### ISSUE THREAD

**Issue by yakovsh** [roots/docs#338]
### Terms

- [X] I have read the [guidelines for Contributing to Roots Projects](https://github.com/roots/.github/blob/master/CONTRIBUTING.md)
- [X] This request is not a duplicate of an existing issue
- [X] I have read the [docs](https://roots.io/docs/) and followed them (if applicable)
- [X] I have seached the [Roots Discourse](https://discourse.roots.io/) for answers and followed them (if applicable)
- [X] This is not a personal support request that should be posted on the [Roots Discourse](https://discourse.roots.io/) community

#### What's wrong?
Unable to run Trellis on MacOS

#### What insights have you gained?
The NFS mount type doesn't work by default on MacOS because of encrypted folders. That is in the docs but the solution is not.

#### Possible solutions
The solution is to change "vagrant_mount_type" in vagrant.default.yml file from "nfs" to "virtualbox" on MacOS. This should be mentioned in the docs.

### Steps To Reproduce

1. Use a MacOS machine with encrypted drive.
2. Try to install and run Trellis.

### Expected Behavior

Works

### Actual Behavior

Doesn't work

### Versions

1.12.0

**Comment by swalkinshaw** [comment]
Yeah this sounds like a useful addition. I'd encourage you to make this change yourself and contribute that improvement

**Comment by juslintek** [comment]
I'd suggest to replace trellis with ddev for local development unless you want to deploy whole trellis infra to staging and production.

**Comment by retlehs** [comment]
@juslintek plenty of folks in this community are using trellis even for just deployment purposes - there's a lot of flexibility with trellis. people should use whatever tool works best for them 😄 

...but coming on here and mentioning a different tool is not relevant to this issue or helpful whatsoever. there's no need to chime in with what your preferred tool is when someone is asking for help on something unrelated to what you're talking about.

**Comment by juslintek** [comment]
Why you see evil in this, when single underlying intention is to help. P.S. You mention DDEV in your docs yourself, over here: https://docs.roots.io/bedrock/master/local-development/

Sometimes changing tool is faster, than getting stuck with same one for weeks.

Maybe If you're using M1 then I guess it might be the issue, because I've had issue running trellis image on M1 so switched to docker. `bento/ubuntu-20.04` (or maybe you have to buy parallels in order to use it) image compatible with arm64 should be released, but is not yet available in cloud https://github.com/chef/bento/issues/1344

**Comment by retlehs** [comment]
Closing this out as Trellis has dropped Vagrant entirely https://roots.io/trellis-drops-vagrant-support-in-favor-of-lima-vms/

**Comment by retlehs** [comment]
@juslintek I suggest DDEV to folks all the time, but lots of us are using Trellis locally and happy. No one is forcing you to use it... I don't even use it for local dev on some of my WP projects

Local dev for Trellis is also in a much better place now with Lima
